blueflood-core/src/test/java/com/rackspacecloud/blueflood/types/MaxValueTest.java

```diff
@@ -32,97 +39,375 @@ public void setUp() {
     }
 
     @Test
-    public void testMaxValueForDoubleMetrics() throws IOException {
-        for (double val : TestData.DOUBLE_SRC) {
-            max.handleFullResMetric(val);
-        }
-        Assert.assertTrue(max.isFloatingPoint());
-        Assert.assertEquals(90.48232472545334, max.toDouble(), 0);
+    public void fullResInitialDoubleSetsValue() {
+
+        // when
+        max.handleFullResMetric(123.45d);
```

Comment:
Ah gotcha. No, it's ok. I understand now. It's really not worth spending anymore time. :-)